src/dispatch/plugins/dispatch_slack/case/interactive.py

```diff
@@ -1748,6 +1927,7 @@ def handle_report_submission_event(
         case_type=case_type,
         dedicated_channel=True,
         reporter=ParticipantUpdate(individual=IndividualContactRead(email=user.email)),
+        assignee=ParticipantUpdate(individual=IndividualContactRead(email=assignee_email)),
```

Comment:
@wssheldon if `assignee_email` is `None`, will this throw an validation error?

Reply:
It does:

```bash
    assignee=ParticipantUpdate(individual=IndividualContactRead(email=None)),
                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for IndividualContactRead
```

I don't know that it *can be* `None` though. You must select an Assignee to proceed no matter what. In the event that it is, what do you think we should default to (assuming it gets that far)? Having a hard time coming up w/ that myself.

Reply:
Line 1916 sets `assignee_email` to `None`. If we're sure that assignee will be selected, could we remove the `if` statement from line 1917?

Reply:
Makes sense, made the change [here](https://github.com/Netflix/dispatch/pull/5197/files#r1759887243)